app/(home)/page.tsx

```
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Button } from '@/components/ui/button'
+import { Input } from '@/components/ui/input'
+import { Progress } from '@/components/ui/progress'
+import { Textarea } from '@/components/ui/textarea'
```

ESLint

[error] 'Textarea' is defined but never used.

(@typescript-eslint/no-unused-vars)